vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/java/com/vaadin/flow/component/spreadsheet/client/CellData.java

```diff
@@ -13,13 +13,16 @@
  * #L%
  */
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+
 import java.io.Serializable;
 
 @SuppressWarnings("serial")
 public class CellData implements Serializable {
 
     public int row;
     public int col;
+    @JsonInclude
```

Comment:
Does this mean that `cd.value` property can never be `null`? In that case, the following branch would never get entered.
```java
 if (cd.value == null) { 
     cachedCellData.remove(key); 
 } else { 
```

Reply:
I don't know exactly on which conditions `CellData.value` can become `null` on the server side, so to test it out, I commented all code writing values to `cd.value` and checked the output of the serialization:

```json
[
	{"row":1,"col":5,"value":null,"cellStyle":"cs7"},
	{"row":1,"col":6,"value":null,"cellStyle":"cs7"},
	{"row":1,"col":7,"value":null,"cellStyle":"cs7"},
]
```

So, it will send the property serialized with the value as `null` in this case and the mentioned branch would be reached.